Dockerfile.s3

```diff
@@ -13,7 +13,7 @@ RUN --mount=type=cache,target=/var/cache/apt \
         # Install dependencies to add additional repos
         apt-get install -y --no-install-recommends \
             # Runtime packages (groff-base is necessary for AWS CLI help)
-             ca-certificates curl make openssl
+             ca-certificates curl make openssl git
```

Comment:
I think we might not need `git` anymore, because we use `SETUPTOOLS_SCM_PRETEND_VERSION_FOR_LOCALSTACK_CORE` instead now.